app/views/partials/_top.html.haml

```diff
@@ -21,7 +21,10 @@
             %li.page_item.page-item-2
               = link_to 'Mer', 'http://sverigeshundforetagare.se/'
             %li.page_item.page-item-2
-              = link_to 'Apply for membership', new_membership_path
+              - if current_user.try(:membership_applications).try(:any?) && current_user.try(:membership_applications).try(:last).try(:persisted?)
```

Comment:
Lets have it like that for now. We can ignore the hound barks until we have figured out a better setup, and then make a PR with that change.